docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -0,0 +1,1249 @@
+      can extend the list of Roxies by either redefining and overriding the
+      list of Roxies, or copying the existing list and adding another entry to
+      it.</para>
+
+      <para>Again, the order the values are merged is the same in which they
+      are issued on the command line.</para>
+
+      <sect3 id="using_the_set_flag">
+        <title>command line options</title>
+
+        <para>If the <emphasis>--set</emphasis> flag is used on helm install
+        or helm upgrade, those values are simply converted to YAML on the
+        client side.</para>
+
+        <para>You can specify the -f flag multiple times. The priority will be
+        given to the last (right-most) file specified. <programlisting>$ helm install myhpcc hpcc/hpcc -f myvalues.yaml -f override.yaml</programlisting></para>
+
+        <para>For the above example, if both myvalues.yaml and override.yaml
+        contained a key called 'Test', the value set in override.yaml would
+        take precedence.</para>
+      </sect3>
+
+      <sect3 id="REPLICAS_">
+        <title>Replicas</title>
+
+        <para>Replicas: defines how many pods (nodes) come up by replicas: how
+        many pods run to balance a load. To illustrate, if you have a 1 node
+        Roxie and set replicas to 2 you would have 2, 1 node Roxies, the
+        second a replica of the first.</para>
+      </sect3>
+
+      <sect3 id="RESOURCES_ValuesYAML">
+        <title>Resources</title>
+
+        <para>Most all components have a resources section which defines how
+        many resources are assigned to that component. Resources and resource
+        definitions are recommended for a live production cloud system. In the
+        stock delivered values files, the resources: sections are there for
+        illustration purposes only, and are commented out so as not to cause
+        any issues with some docker desktop deployments, etc. Any cloud
+        deployment that will be performing a non-trivial function, these
+        values should be properly defined with adequate resources for each
+        component, in the same way you would allocate adequate physical
+        resources in a data center. Improper resource definition can result in
+        running out of memory and Kubernetes eviction, since the system will
+        use unbound amounts of memory nodes will get overwhelmed and
+        Kubernetes will evict. Therefore if your deployment is seeing frequent
+        evictions, you may want to adjust your resource allocation.</para>
+
+        <para><programlisting>  #resources:
+  #  cpu: "1"
+  #  memory: "4G"
+</programlisting>Every component should have a resources entry, but some
+        components such as Thor has multiple resources. The manager: worker,
+        eclagent components all have different resource requirements.</para>
+      </sect3>
+
+      <sect3 id="TAINTS_TOLERATIONS_PLACEMENTS">
+        <title>Taints, Tolerations, and placements</title>
```

Comment:
This is a bit of an obscure topic to address this early in the document.  Ok if it is dealing with an expanded section describing common component properties, but there are probably some others that should be here if so.

Reply:
I wonder if the document would read better if each of these sections were included in the description of the standard values.yaml.